local.yml

```diff
@@ -17,6 +17,7 @@ services:
     environment:
       - "discovery.type=single-node"
       - "transport.host=127.0.0.1"
+      - "xpack.security.enabled=false"
```

Comment:
docker was complaining about lacking a license key to keep evaluating xpack; this disables xpack, which isn't needed for this project anyway